application/org.openjdk.jmc.flightrecorder.flameview/src/main/js/flameviewColoring.js

```diff
@@ -117,9 +117,21 @@ const createHslColorString = function(h,s,l){
 };
 
 const colorCell = function (d) {
-	return colorByPackage(d.data.p);
+	if (textToSearch !== "" && (evaluateSearchElement(d.data.p) || evaluateSearchElement(d.data.n))){
+		return "magenta";
+	} else {
+		return colorByPackage(d.data.p);
+	}
 };
 
+const evaluateSearchElement = function(text){
+	if((text !== undefined && text.toLowerCase().includes(textToSearch))){
```

Comment:
formatting: space after `if`
formatting: looks like there's an extra pair of brackets on the if-statement
formatting: space before {